src/scripts/main.js

```diff
@@ -6,3 +6,38 @@ const people = require('./lib/people');
 console.log(people); // you can remove it
 
 // write your code here
+const table = document.querySelector('.dashboard');
+
+let tableRow = '';
+const tableHeader = `
+  <tr>
+    <th>Name</th>
+    <th>Gender</th>
+    <th>Born</th>
+    <th>Died</th>
+    <th>Age</th>
+    <th>Century</th>
+  </tr>
+`;
+
+for (const person of people) {
+  const personAge = person.died - person.born;
+  const personCentury = Math.ceil(person.died / 100);
+  const personGender = person.sex === 'm' ? 'Male' : 'Female';
+
+  tableRow += `
+    <tr>
+      <td>${person.name}</td>
+      <td>${personGender}</td>
+      <td>${person.born}</td>
+      <td>${person.died}</td>
+      <td>${personAge}</td>
+      <td>${personCentury}</td>
+    </tr>
+  `;
+
+  table.innerHTML = `
+  ${tableHeader}
+  ${tableRow}
+  `;
```

Comment:
this code isn't optimal because it will overwrite the `table.innerHTML` in each cycle of the loop so it would be better to move it out of the loop

Reply:
Noted it, made the changes; should I push it if it's already approved?

Reply:
> this code isn't optimal because it will overwrite the `table.innerHTML` in each cycle of the loop so it would be better to move it out of the loop

Noted it, made the changes; should I push it if it's already approved?